help/destinations/catalog/personalization/algolia-user-segments.md

```diff
@@ -0,0 +1,147 @@
+
+## Prerequisites {#prerequisites}
+
+>[!IMPORTANT]
+>
+>* To connect to the destination, you need the **[!UICONTROL View Destinations]** and **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+>* To export *identities*, you need the **[!UICONTROL View Identity Graph]** [access control permission](/help/access-control/home.md#permissions). <br> ![Select identity namespace highlighted in the workflow to activate audiences to destinations.](/help/destinations/assets/overview/export-identities-to-destination.png "Select identity namespace highlighted in the workflow to activate audiences to destinations."){width="100" zoomable="yes"}
+
+## Supported audiences {#supported-audiences}
+
+This section describes which type of audiences you can export to this destination.
+
+| Audience origin | Supported | Description |
+|---------|---------|----------|
+| [!DNL Segmentation Service] | ✓ | Audiences generated through the Experience Platform [Segmentation Service](../../../segmentation/home.md). |
+
+{style="table-layout:auto"}
+
+## Export type and frequency {#export-type-frequency}
+
+Refer to the table below for information about the destination export type and frequency.
+
+| Item | Type | Notes |
+|---------|----------|---------|
+| Export type | **[!DNL Profile request]** | You are requesting all the audiences that are mapped in the Algolia User Segments destination for a single profile.|
+| Export frequency | **[!UICONTROL Streaming]** | Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on audience evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).|
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+>[!IMPORTANT]
+>
+>To connect to the destination, you need the **[!UICONTROL View Destinations]** and **[!UICONTROL Manage and Activate Dataset Destinations]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the destination configuration workflow, fill in the fields listed in the two sections below.
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+#### New account {#new-account}
```

Comment:
No need for the `#### New account {#new-account}` heading. Please remove it and use the format in the template (https://experienceleague.adobe.com/en/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template#authenticate) exactly as it is there, just replace the values with those applicable to Algolia.